ProcessMaker/ImportExport/Exporters/ExporterBase.php

```diff
@@ -65,6 +65,13 @@ public static function modelFinder($uuid, $assetInfo)
         $matchedBy = null;
         $baseQuery = $class::query();
 
+        $model = new $class;
+
+        // If table does not exists for model, continue.
+        if (!Schema::hasTable($model->getTable())) {
```

Comment:
@agustinbusso could you change the `$model` variable name so we don't confuse it with the model we use later in the code? Looks like you could also do this:
```suggestion
        // If table does not exists for model, continue.
        if (!Schema::hasTable($baseQuery->getModel()->getTable())) {
```